app/src/test/java/com/futurice/freesound/feature/search/DefaultSearchDataModelTest.java

```diff
@@ -171,10 +167,10 @@ public void getSearchResults_emitsEmptyList_whenCleared() {
 
     @Test
     public void clear_emitsSingleUnit() {
-        TestSubscriber<Void> ts = defaultSearchDataModel.clear().test();
-
-        ts.assertNoErrors()
-          .assertComplete();
+        defaultSearchDataModel.clear()
+                              .test()
+                              .assertNoErrors()
```

Comment:
This one is a completable, you already assert it to be completed. In that case it won't have an error